osu.Game/Graphics/OsuColour.cs

```diff
@@ -75,9 +75,12 @@ public Color4 ForHitResult(HitResult result)
         {
             switch (result)
             {
+                case HitResult.IgnoreMiss:
                 case HitResult.SmallTickMiss:
-                case HitResult.LargeTickMiss:
+                    return Orange1;
```

Comment:
Just to confirm - this will also apply to `ColourHitErrorMeter`, skinnable judgement counter, etc.; are we OK with this?

Reply:
Yeah I think so!